ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -94,6 +100,9 @@ public void onLayoutChange(
   private @Nullable ReactViewBackgroundDrawable mReactBackgroundDrawable;
   private @Nullable OnInterceptTouchEventListener mOnInterceptTouchEventListener;
   private boolean mNeedsOffscreenAlphaCompositing = false;
+  private float mLastExplicitlySetOpacity = 1.f;
+  private String mLastExplicitlySetBackfaceVisibility;
```

Comment:
Why is this "last explicitly set"? Isn't backface visibility always explicitly set? Can't we call it just `mBackfaceVisibility` and default to visible?